### PR TITLE
Feat:added product list

### PR DESCRIPTION
**Pull request by Rohanmond**
Screenshot 2022-02-10 at 12 59 17 PM
added product list

### CODE REVIEW

File: pages/productList.html [hunk 1]
```diff
@@ -0,0 +1,499 @@
...
+                alt=""
+              />
+            </a>
+          </div>
+        </div>
+        <div class="nav-mid nav-desktop">
+          <input
+            placeholder="search"
+            class="nav-search brd-rd-semi-sq nav-text-input"
+            type="text"
```
Comment (ravikumar1002) [inline pages/productList.html]: You can also use ``` type = "search"```

File: pages/productList.html [hunk 2]
```diff
@@ -0,0 +1,499 @@
...
+                  id="2star-radio"
+                  value="2star"
+                />
+                <label for="2star-radio">2 stars &#38; above</label>
+              </div>
+              <div class="productlist-input-item-hz">
+                <input
+                  type="radio"
+                  name="rating-radio"
+                  id="1star-radio"
```
Comment (ravikumar1002) [inline pages/productList.html:157]: I am not sure but I think , id doesn't start with number

File: pages/productList.html [hunk 3]
```diff
@@ -0,0 +1,499 @@
...
+    <title>FashiFy | Product List</title>
+
+    <link rel="stylesheet" href="../style.css" />
+  </head>
+  <body>
+    <div class="productlist-container">
+      <nav class="navigation">
+        <div class="nav-left">
+          <div class="nav-logo-container">
+            <a href="">
```
Comment (ravikumar1002) [inline pages/productList.html]: One suggestion, instead of Blank href you can use "#" value

**Review by ravikumar1002 — APPROVED** [review]
everything looks good, Rohan

File: pages/productList.html (hunk 3 above)
Comment (Rohanmond) [reply pages/productList.html]: It is fixed now.Thanks for your review

File: pages/productList.html (hunk 2 above)
Comment (Rohanmond) [reply pages/productList.html:157]: It is for the label only so I put like that

File: pages/productList.html (hunk 1 above)
Comment (Rohanmond) [reply pages/productList.html]: Thanks for your suggestion

**Review by Shah-Arsalan — APPROVED** [review]

**Pull request merged**